sample/build.gradle

```diff
@@ -16,6 +16,13 @@ android {
         versionName "1.0"
     }
     buildTypes {
+        debug {
+            // for testing proguard
```

Comment:
It increases the compile time. But you're right, this sample is so small it doesn't matter